frontend/src/components/worksheets/items/TableItem/TableItem.js

```diff
@@ -115,6 +116,19 @@ class TableItem extends React.Component<{
         });
     };
 
+    showCheckedBundleRowsContents = () => {
+        let item = this.props.item;
+        let rows = [...Array(item.rows.length).keys()].filter((item, index) => {
```

Comment:
Why can't we just use `item.bundles_spec.bundle_infos` here like we did in the `copyCheckedBundleRows` function?

Reply:
basically, it is because the filtering of the checked bundles are different in `copyCheckedBundleRows` and `showCheckedBundleRowsContents`. In `copyCheckedBundleRows`, we only care about the bundle itself, while in `showCheckedBundleRowsContents`, we also care about the raw index of each checked bundle.